### ISSUE TITLE
Unable to detect expiry date

### ISSUE THREAD

**Issue by Tronicnotes** [getbouncer/cardscan-android#113]
There still appears to be issues reading the credit card expiry values.  Card number detection works well, but the expiry won't be picked up at all.  Have tried a variety of valid cards and different devices, none of which can detect the expiry.  These cards have all the details on the front.

For reference, this valid testing card image was used https://miro.medium.com/max/1778/0*KcSus6ZmvPhsKWnS

Similar to #32

**Comment by kingst** [comment]
tl;dr: it's only on iOS for now, but coming to Android soon.

details: Running deep neural nets on Android is really really hard. We have a model that pulls out the card number and expiry on iOS and works well due to the hardware acceleration for ML on their phones. However, when we shipped that same model on Android it struggled on low end devices, which are important. We completely revamped the way we do our ML pipeline to make sure that we could get low end devices as well, but had to simplify the model a bit to accomplish this, so we had to drop expiry. However, we have a plan to bring it back and are actively working on it now. I do expect to have expiry back on Android in the next month or two.

The most surprising part of this is that for card numbers our ML on Android outperforms our iOS system despite the massive hardware advantage that iOS has! We'll also port these improvements to iOS in time as well.

**Comment by awushensky** [comment]
I'm going to close this issue since it's a duplicate of https://github.com/getbouncer/cardscan-android/issues/82.
